docs/spec.md

```diff
@@ -2244,26 +2263,29 @@ the XLA compiler. In the future, we are planning to unify this metadata
 Performs element-wise division of dividend `lhs` and divisor `rhs` tensors and
 produces a `result` tensor. Depending on the element type, does the following:
 
-* For integers: integer division.
+* For integers: integer division which produces the algebraic quotient with any
+  fractional part discarded.
 * For floats: `division` from IEEE-754.
 * For complex numbers: complex division.
+* For quantized types:
+  * `dequantize_op_quantize(divide, lhs, rhs, type(result))`.
 
 #### Inputs
 
-| Label | Name  | Type                                              | Constraints |
-|-------|-------|---------------------------------------------------|-------------|
-| (I1)  | `lhs` | tensor of integer, floating-point or complex type | (C1)        |
-| (I2)  | `rhs` | tensor of integer, floating-point or complex type | (C1)        |
+| Label | Name  | Type                                                                              | Constraints |
+|-------|-------|-----------------------------------------------------------------------------------|-------------|
+| (I1)  | `lhs` | tensor of integer, floating-point, or complex type or per-tensor quantized tensor | (C1)        |
+| (I2)  | `rhs` | tensor of integer, floating-point, or complex type or per-tensor quantized tensor | (C1)        |
```

Comment:
I don't think we're using commas for similar situations anywhere else. Defer to a follow-up PR?

Reply:
makes sense.